doc/content/toolstack/features/MVD/index.md

```diff
@@ -0,0 +1,347 @@
+'++
+title = "Multi-version drivers"
++++
+
+Linux loads device drivers on boot and every device driver exists in one
+version. XAPI extends this scheme such that device drivers may exist in
+multiple variants plus a mechanism to select the variant being loaded on
+boot. Such a driver is called a multi-version driver and we expect only
+a small subset of drivers, built and distributed by XenServer, to have
+this property. The following covers the background, API, and CLI for
+multi-version drivers in XAPI.
+
+## Variant vs. Version
+
+A driver comes in several variants, each of which has a version. A
```

Comment:
It can be useful when you want to update (backport) a driver (e.g for newer hardware or some risky fixes) without the risk of introducing regressions.

Reply:
You mean updating a driver to a variant, but that update itself is safe? How does it achieve that safety?